src/crypto/muhash.cpp

```diff
@@ -0,0 +1,307 @@
+// Copyright (c) 2017 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <crypto/muhash.h>
+
+#include <crypto/chacha20.h>
+#include <crypto/common.h>
+
+#include <assert.h>
+#include <stdio.h>
+
+#include <limits>
```

Comment:
No need for a blank line here. These are all standard library includes and can be grouped together (as well as using the C++ header names):

```suggestion
#include <cassert>
#include <cstdio>
#include <limits>
```

Reply:
Done